cpp/tests/test_utils.h

```diff
@@ -62,6 +62,23 @@ extern "C" {
 }
 #endif
 
+#define NCCLCHECK(cmd) do {                                   \
+    ncclResult_t nccl_status = cmd;                           \
+    if (nccl_status!= ncclSuccess) {                          \
+      printf("NCCL failure %s:%d '%s'\n",                     \
```

Comment:
Should this fail the test?  It looks like it's just going to print a message to stdout.

Reply:
yes